results/v1.19/known-test-failures.md

```diff
@@ -0,0 +1,12 @@
+# Known Test Failures for v1.19
+
+This page aims to summarize the causes of known test failures when running
+this version of Kubernetes conformance tests against a Supervisor cluster
+running vSphere Pods.
+
+- Test: `Kubectl cluster-info should check if Kubernetes master services is included in cluster-info`
```

Comment:
Is this covered here ? https://github.com/vmware-tanzu-experiments/vsphere-supervisor-for-kubernetes-conformance/blob/main/known-test-failures.md

Reply:
Thanks for taking a look.
It is not. 
The reason is that this test fails only for 1.19 and because of a specific reason for this test (string was updated).
As I understand https://github.com/vmware-tanzu-experiments/vsphere-supervisor-for-kubernetes-conformance/blob/main/known-test-failures.md is for common failures for Supervisor in general